.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Roles of a PR: Author, Reviewer, and Assignee
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
+      A Reviewer reviews the change of a PR,
+      leaves comments for the Author to address,
+      and approves the PR when the change looks good.
+    * An Assignee must be a maintainer, who monitors the progress of the PR,
```

Comment:
Maintainer should be caps, add link to https://github.com/PowerShell/PowerShell/tree/master/docs/maintainers